project/Team.py

```diff
@@ -1,173 +1,532 @@
 import unittest
+import datetime
+import time
 from abc import ABC, abstractmethod
-from GameMaker import UserABC
 
+from Landmark import LandmarkFactory
+from Team import TeamFactory
+
+
+class GameInterface(ABC):
```

Comment:
uhhh, This is Team.py... Why are we declaring a GameInterface here?

Reply:
I have no clue, this was included in my pull lol